auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -39,15 +38,18 @@ private static String newRequestId() {
                         .build())) {
             UmsClient client = new UmsClient(channelWrapper.getChannel(), actorCrn);
             String requestId = newRequestId();
```

Comment:
Are we planning to switch to using the x-cdp-request-id for this?

Reply:
Yes, we have a story for that, and we will be propagating the request-id we recieve in the header, but that will be in another PR.